src/core/generator/snapshots/tailcall__core__generator__generator__test__read_all_with_different_domain_rest_api_gen.snap

```diff
@@ -45,37 +44,38 @@ type T18 {
   weight: Int
 }
 
-type T19 {
-  limit: Int
-  products: [T18]
-  skip: Int
-  total: Int
-}
-
-type T2 {
-  body: String
-  id: Int
-  title: String
-  userId: Int
-}
-
-type T3 {
-  depth: Int
-  height: Int
-  width: Int
+type Query {
+  f1(p1: Int!): [RootType1] @http(baseURL: "https://jsonplaceholder.typicode.com", path: "/posts/{{.args.p1}}/comments")
```

Comment:
This still seems incorrect to me. The TypeNameGenerator, should have used `f1` which is the name of the field in Query for its type name.